### ISSUE TITLE
Add support Multiple X, Y, Z Boundries

### ISSUE THREAD

**Issue by ariaamz** [bjadidi/RSD#2]
Most Users, would like to provide multiple x y and z values to be processed.

**Comment by bjadidi** [comment]
Yeah, We should also add a CLI UI for users to enter corresponding X, Y, and Z values
